TROUBLESHOOTING.md

```diff
@@ -0,0 +1,70 @@
+# Troubleshooting
```

Comment:
Do we have a section in the official documentation about Troubleshooting? if not, we should add this guide also there.

Reply:
@andreaangiolillo By official documentation do you mean on https://registry.terraform.io/ ? 

If so, currently there's no section for Troubleshooting. @Zuhairahmed can you please confirm adding this to our provider on https://registry.terraform.io/ ?

Reply:
> By official documentation do you mean on https://registry.terraform.io/ ?

Yes, this one https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs

Reply:
yes this is a good add. we can also move out troubleshooting guidance from our ReadMe in repo to this new centralized location. suggest we also added a "Troubleshooting" section on on our GitHub Readme that simply links to website/docs/TROUBLESHOOTING.md so everyone know to go this new location

Reply:
This has been updated